k8s/sandbox/cluster-00/probate-corn-job/business-service.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-corn-job-test
+  namespace: probate
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/ignore: "false"
+spec:
+  releaseName: probate-corn-job-test-1
+  rollback:
+    enable: true
+  chart:
+    image: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: job
+    version: 0.5.2
+  values:
+    job:
+      image: 'hmctspublic.azurecr.io/docker-curl:latest'
+      schedule: "0/4 0 * * *"
+      aadIdentityName: probate
+      args: 
```

Comment:
I don't think this is valid argument for job chart
https://github.com/hmcts/chart-job#configuration